impl/client/src/main/javascript/web/dojo/pentaho/common/FilterDialog.js

```diff
@@ -33,6 +33,13 @@ define(["dojo/_base/declare", "dijit/_WidgetBase", "dijit/_TemplatedMixin", "dij
 
   buttons: ['Ok_txt','Cancel_txt'],
 
+  ORIENTATION_LANDSCAPE: 0,
+  ORIENTATION_PORTRAIT: 1,
+  orientation: undefined,
+  _onCancelCallback: undefined,
```

Comment:
What are these properties? Where are they used?

Reply:
Added by mistake. Will be removing them.